tests/index.js

```diff
@@ -120,16 +120,19 @@ describe('Thorn', () => {
 
                         expect(request.url).to.contain(fixture.module);
                         expect(request.method).to.equal('POST');
-                        expect(request.data).to.eql(fixture.attributes);
+                        expect(request.data).to.contain(fixture.attributes);
 
                         expect(this.req.headers['x-thorn']).to.equal('Fixtures');
 
                         return constructBulkResponse({
                             _module: fixture.module,
                             id: 'TestId1',
-                            name: fixture.attributes.name,
-                            testField1: fixture.attributes.testField1,
-                            testField2: fixture.attributes.testField2,
```

Comment:
Returned values should be based on data retrieved from the request data, not the fixture itself, we specially need to do that for attributes that were not initially defined at the fixture level, e.g: testField[3|4|5]